storage-management/versioning-limits.md

```diff
@@ -135,3 +133,24 @@ To remove vesion limits, **open the Storage Metrics report**.
 ![Storage Versioning Limits - Remove Manually Applied Limits](../.gitbook/assets/storage-versioning-limits-remove-manually-applied.png)
 
 ![Storage Versioning Limits - Confirm Removal](../.gitbook/assets/storage-versioning-limits-remove-manually-applied-confirmation.png)
+
+## Manual Cleanup after Storage Versioning Limits
+
+In some instances, even after setting up storage versioning limits, you **might need to perform a manual storage cleanup**. 
+
+The **Storage Versioning Limits only apply to new file versions created after the policy is enabled**. Older file versions are not automatically deleted **unless a manual cleanup is performed first**. For example:
+
+* If you only set a **version count limit** (for example, keep 100 versions), and run the manual cleanup once **before** enabling the policy, the **manual cleanup removes all excess file versions** and from then on the **cleanup is handled automatically** for new versions
+
+* If you are using the version count limit in combination with the time limit, the feature **only applies the time-based deletion to versions created after the policy was set**, so versions that were not old enough at the time of the initial cleanup will not be deleted later, unless you repeat manual cleanup over time
+  * If you are using the **Time Limit**, we recommend **repeating manual cleanup periodically during the first year**.
+
+
+{% hint style="info" %}
+**Please note**: Even with storage cleanup automation turned on, you might notice that Potential Savings are still growing. This is expected due to:
+* **Some older file versions may remain if they were not manually deleted during the initial cleanup**
+* **There is a delay in storage sync, which can happen in large environments** which means that file versions that were already deleted in SharePoint could still appear in reports until the next sync is completed
+  * Depending on the size of the environment, the storage sync can take **up to a month**
+  * If you have a large tenant, you can use **on-demand sync to target specific environments or sites** instead of waiting for the full sync cycle to be completed
```

Comment:
Fali objašnjenje što je on-demand storage sync i gdje se može pokrenuti. 
Ovo je ispalo jer se naknadno uključio feature nakon releasea.
Predlažem da se doda novi article - Storage sync u Storage Management sekciju pa se tamo objasni kada se odrađuje automatski Storage sync, kako ručno pokrenuti ovaj on-demand storage sync i ostalo što treba.